count-stats/src/main/java/uk/ac/ebi/eva/countstats/service/CountService.java

```diff
@@ -0,0 +1,23 @@
+package uk.ac.ebi.eva.countstats.service;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+import uk.ac.ebi.eva.countstats.model.CountDto;
+import uk.ac.ebi.eva.countstats.repository.CountRepository;
+
+import java.util.List;
+
+@Service
+public class CountService {
+    @Autowired
```

Comment:
My personal preference is to use [constructor injection](https://docs.spring.io/spring-boot/docs/current/reference/htmlsingle/#using.spring-beans-and-dependency-injection), although I know we use field injection in many repositories.

Reply:
Removed field injection and added constructor injection in both Controller and Service